internal/utils/file-utils.go

```diff
@@ -0,0 +1,49 @@
+package utils
```

Comment:
This should have another name. `utils` package are very badly name (utils doesn't mean anything — see https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common).